### DISCUSSION TITLE
New Entities from Cerbo GX

### DISCUSSION THREAD

**Discussion by askpatrickw** [sfstar/hass-victron#116] [Q&A] [answered]
Since I originally setup HASS-Victron, I have added a new USB GPS device to my Cerbo GX. I can see the GPS is working in the Victron UI, I can use pymodbus directly to the Cerbo and see the GPS data via the registers, but there are no new entities showing up in HA. 

Is "rediscovery" for new entities something that is natively part of HA and I hit a bug or should HASS Victron be handling that?

ty!

**Comment by andzie82** [comment]
Have you tried to reconfigure the add on?

**Reply by askpatrickw** [reply, marked as answer]
Ah, thanks for the reply. I didn't see that checkbox down there!
Let me make it more clear for the next reader.

1. In devices and services select Victron.
2. Click reconfigure on the hub you created the first time you set everything up.
3. Scroll down on the configuration page (you can't see it till you scroll to the bottom) select "Rescan available devices. This will rescan all available devices".
4. Click submit on the bottom of the form.

**Comment by hvestbo** [comment]
Have the same issue.
Clicking configure and rescan, but cant find any entities of the gps
